packages/pipeline/src/parsers/github/repo.ts

```diff
@@ -0,0 +1,23 @@
+import { GithubRepoResponse } from '../../data_sources/github';
+import { GithubRepo } from '../../entities';
+
+/**
+ * Converts a Github response from the API into an GithubRepo entity.
+ * @param rawRepo A Github response from the API into an GithubRepo entity.
+ */
+export function parseGithubRepo(rawRepo: GithubRepoResponse, observedTimestamp: number): GithubRepo {
+    const parsedRepo = new GithubRepo();
+    parsedRepo.observedTimestamp = observedTimestamp;
+    parsedRepo.name = rawRepo.name;
```

Comment:
I wonder if we should use `rawRepo.full_name` here, so that we retain the option to track other orgs in the future, for example to compare our developer response time to other competing projects.

Reply:
Makes sense, I'll change it